plugins/sqlfluff-templater-dbt/src/dbt_templater/__init__.py

```diff
@@ -0,0 +1,10 @@
+"""Defines the hook endpoints for the dbt templater plugin."""
+
+from dbt_templater.templater import DbtTemplater
+from sqlfluff.core.plugin import hookimpl
+
+
+@hookimpl
+def get_templaters():
+    """Get templaters."""
+    return [DbtTemplater]

```

Comment:
if I understand correctly @alanmcruickshank, this PR makes templaters pluggable in general?
If that's the case, maybe it's worth reflecting in the PR title/description so it appears in the CHANGELOG, and also updating the plugin Documentation?

I've seen a few issues in the past with people having some very custom ways of templating SQL, this PR might address those too